### PR TITLE
Data entrypoint

### PR DESCRIPTION
**Pull request by sjaks**
[This code](https://github.com/Alylamppuvegaanit/discord-voice-bot-web/compare/data?expand=1#diff-a9259d948d9e0ef9c88b9cd5c88bd070f53ee80c503a4422b8665721fe0df282R43-R62) is vey shitty. Good algorithm needs to be coded and I haven't taken tiraka yet so maybe some one can fix this 🤔

### CODE REVIEW

**Comment by sjaks** [comment]
Otherwise code very good, it works 😃

File: discord-voice-bot-web.js [hunk 1]
```diff
@@ -1,42 +1,75 @@
...
+    let updatedPlaylists = req.body;
+
+    /*
+    Currently this part relies on the front-end to have a copy of all
+    the desired information. TODO: do not replace all data with one
+    coming from the front-end but process the diff of playlists here
+    and combine the new data with old data. Below some experimental stuff.
+
+    let savedPlaylists = JSON.parse(fetchPersistent());
+    let combinedNewData = savedPlaylists.concat(updatedPlaylists)
```
Comment (vilukissa68) [inline discord-voice-bot-web.js]: This is very bad idea

Comment (vilukissa68) [reply discord-voice-bot-web.js]: Please make better one

**Review by vilukissa68 — CHANGES_REQUESTED** [review]

**Comment by sjaks** [comment]
Jaa ok

File: frontend/src/PlayListTable.js [hunk 2]
```diff
@@ -102,25 +175,21 @@ function PlayListTable() {
 
     // Do basic validation and check whether a playlist is selected
     if (newSongName.length > 5 && activePlaylist.length > 2) {
-      const tmp = songRows.concat({ title: newSongName, id: songRows + 1 });
+      const tmp = songRows.concat({ title: newSongName, id: songRows.length + 1 });
```
Comment (vilukissa68) [inline frontend/src/PlayListTable.js:178]: Consider generating ids by random seeds

**Review by vilukissa68 — APPROVED** [review]
This code is lezz sitty than the last code. Good job!

File: frontend/src/PlayListTable.js (hunk 2 above)
Comment (sjaks) [reply frontend/src/PlayListTable.js:178]: Maybe one day yes

**Pull request merged**